### ISSUE TITLE
openaire: make DFG grants available

### ISSUE THREAD

**Issue by slint** [zenodo/zenodo#2142]
It seems that grants coming from DFG are [available in OpenAIRE BETA](https://beta.explore.openaire.eu/search/find/projects?f0=q&fv0=&page=1&size=10&funder=%22dfgf________%253A%253ADFG%257C%257CDeutsche%2520Forschungsgemeinschaft%257C%257CDFG%22), but not in production (Alessia or someone else might be able to help with that).

**Comment by alessiabardi** [comment]
Yes, it is correct. I will give you a note when they'll be available also in the production system.
